tests/test_views.py

```diff
@@ -147,11 +147,6 @@ def test_location_detail(client, user_staff, location):
 def test_survey_list_edit(client, user_staff, survey_form_entry,
                           survey_form_entry_inactive, survey_form_entry_observational,
                           survey_form_entry_agency_2):
-    # Make sure that the SurveyFormEntry from Agency 2 is not published, to verify
-    # that it doesn't get loaded on the Edit list view.
-    survey_form_entry_agency_2.published = False
-    survey_form_entry_agency_2.save()
```

Comment:
This fixture is initialized with `published=False` so the update here is redundant.